rfcs/0015_Arm_CMSIS-NN_Integration.md

```diff
@@ -78,14 +96,18 @@ primfn(placeholder_1: handle, out_write_1: handle) -> ()
     }
 }
 ```
+In future, target hooks for `relay_to_tir` implemented as part of [Additional Target Hooks] (https://github.com/apache/tvm-rfcs/pull/10) will be used to obtain the above tir for graph with softmax. These hooks provide us with the flexibility to reuse memory planning and much of the TVM's code generation capabilities.
+
+At last, code generator identifies the *tir* extern_call(s) and generates *c* code for softmax with the CMSIS-NN API for softmax int8.
+
+Note: There are no changes required in config.cmake as the CMSIS-NN APIs corresponding to the operators are hard coded. The testing infrastructure links them to the CMSIS-NN library. Execution of the networks works similar to what has been described in [Arm Ethos-U Integration] (https://github.com/apache/tvm-rfcs/pull/11).
```

Comment:
Even the CMSIS-NN is dynamically linked, I would still suggest letting users turn on/off using config.cmake to control source files, facilitate the TVM build time, and the TVM binary size.

Reply:
ACK

Reply:
i think if there are just python-side changes, it's probably okay to avoid config.cmake changes. wdyt?

Reply:
There is '.cc'  source file that generates c code eventually